11.coreboot.u-root.systemboot/linux-4.19.6-linuxboot.config

```diff
@@ -1726,8 +1818,10 @@ CONFIG_FS_MBCACHE=y
 #
 # CD-ROM/DVD Filesystems
 #
-# CONFIG_ISO9660_FS is not set
-# CONFIG_UDF_FS is not set
+CONFIG_ISO9660_FS=y
```

Comment:
you don't need these. did you upgrade the kernel at the same time and use the default options?